test/format/format.sh

```diff
@@ -231,12 +238,24 @@ resolve()
 		# Skip failures we've already reported.
 		[[ -f "$dir/$status" ]] && continue
 
-		# Get the proceess ID, ignore any jobs that aren't yet running.
+		# Get the process ID, ignore any jobs that aren't yet running.
 		pid=`grep -E 'process.*running' $log | awk '{print $3}'`
 		[[ "$pid" =~ ^[1-9][0-9]*$ ]] || continue
 
-		# If the job is still running, ignore it.
-		kill -s 0 $pid > /dev/null 2>&1 && continue
+		# Leave any process waiting for a gdb attach running, but report it as a failure.
+		grep -E 'waiting for debugger' $log > /dev/null && {
```

Comment:
Is `waiting for debugger` the format program output that indicates a test is failed and has not been gdb-attached?

Reply:
`waiting for debugger` is the `format` output that means a process is waiting for `gdb` to attach to it.

Reply:
I'm curious how a format process could be set to a state waiting for gdb attachment. Is it a `format` program argument?

Reply:
It's a `format` build option, `--enable-attach`, which causes certain types of failures to print out a 
 message and then spin, waiting for `gdb` to attach.

Reply:
`waiting for debugger` is actually WiredTiger output, not `format` output. It is a WiredTiger library build option, not specific to `format`.

Reply:
Thanks for clarifying that, @sueloverso.